templates/admin/users_banned.html.twig

```diff
@@ -0,0 +1,85 @@
+    {{- 'users'|trans }} - {{ parent() -}}
+{%- endblock -%}
+
+{% block mainClass %}page-admin-users{% endblock %}
+
+{% block header_nav %}
+{% endblock %}
+
+{% block sidebar_top %}
+{% endblock %}
+
+{% block body %}
+    {% include 'admin/_options.html.twig' %}
+
+    <div class="pills">
+        <menu>
+            <li>
+                <a href="{{ path('admin_users_active') }}"
+                   class="{{ html_classes({'active': is_route_name('admin_users_active')}) }}">
+                    {{ 'admin_users_active'|trans }}
+                </a>
+            </li>
+            <li>
+                <a href="{{ path('admin_users_inactive') }}"
+                   class="{{ html_classes({'active': is_route_name('admin_users_inactive')}) }}">
+                    {{ 'admin_users_inactive'|trans }}
+                </a>
+            </li>
+            <li>
+                <a href="{{ path('admin_users_suspended') }}"
+                   class="{{ html_classes({'active': is_route_name('admin_users_suspended')}) }}">
+                    {{ 'admin_users_suspended'|trans }}
+                </a>
+            </li>
+            <li>
+                <a href="{{ path('admin_users_banned') }}"
+                   class="{{ html_classes({'active': is_route_name('admin_users_banned')}) }}">
+                    {{ 'admin_users_banned'|trans }}
+                </a>
+            </li>
+        </menu>
+    </div>
+
+    {% if(users.haveToPaginate is defined and users.haveToPaginate) %}
+        {{ pagerfanta(users, null, {'pageParameter':'[p]'}) }}
+    {% endif %}
+    <div class="section" id="content">
+        <div class="flex" data-controller="selection">
+            <label class="select">
+                <select data-action="selection#changeLocation">"
+                    <option value="{{ options_url('withFederated', false) }}">{{ 'local'|trans }}</option>
+                    <option value="{{ options_url('withFederated', true) }}" {{ withFederated is same as true ? 'selected' : '' }}>{{ 'federated'|trans }}</option>
+                </select>
+            </label>
+        </div>
+        <table>
+            <thead>
+            <tr>
+                <th>{{ 'username'|trans }}</th>
+                <th>{{ 'email'|trans }}</th>
+                <th>{{ 'created_at'|trans }}</th>
+                <th>{{ 'last_active'|trans }}</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for user in users %}
+                {% if not user.isDeleted %}
+                    <tr>
+                        <td>{{ component('user_inline', {user: user}) }}</td>
+                        <td>{{ user.apId ? '-' : user.email }}</td>
+                        <td>{{ component('date', {date: user.createdAt}) }}</td>
+                        <td>{{ component('date', {date: user.lastActive}) }}</td>
+                    </tr>
+                {% endif %}
+            {% endfor %}
+            </tbody>
+        </table>
+    </div>
```

Comment:
If this is the same in all 4 files you can put it in its own file as well

Reply:
yep, also makes sense!